### PR TITLE
Automation cmdlets bug fixes and updated with Automation SDK version  1.0.1

### PR DESCRIPTION
**Pull request by safeermohammed**
Automation team would request to merge Automation cmdlets bug fixes for 0.8.16 train.
- PR has few bug fixes
- Update to latest Azure Automation nuget package 1.0.1

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@safeermohammed__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (safeerm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by AzureRT** [comment]
Can one of the admins verify this patch?

**Comment by jodoglevy** [comment]
@markcowl @ogail can we have this merged please? We'd like the changes to make it into the next Azure PS drop, and as I understand it the deadline for merges is today.

File: src/ServiceManagement/Automation/Commands.Automation/RequestSettings.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
```
Comment (ogail) [inline src/ServiceManagement/Automation/Commands.Automation/RequestSettings.cs]: Add copyright header

**Comment by safeermohammed** [comment]
Thanks,  Added copy right and moved the file to common folder in the project

File: src/ServiceManagement/Automation/Commands.Automation/RequestSettings.cs (hunk 1 above)
Comment (safeermohammed) [inline src/ServiceManagement/Automation/Commands.Automation/RequestSettings.cs]: Done

**Comment by ogail** [comment]
@azuresdkci add to whitelist

**Pull request merged**